user/oils-for-unix/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "oils-for-unix"
+pkgver = "0.29.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = []
+make_dir = "."
+hostmakedepends = ["autoconf"]
+makedepends = ["readline-devel"]
+pkgdesc = "Unix shell with structured data"
+license = "Apache-2.0"
+url = "https://oils.pub"
+source = f"{url}/download/oils-for-unix-{pkgver}.tar.gz"
+sha256 = "34fdb53f1dbd2b8742ef74cd3a46d87abfcda18d0d645f29762c4fddb9a2b3ac"
+# check: no tests in tarball
+options = ["!check"]
+
+
+def init_configure(self):
+    self.configure_args += [
+        "--readline",
+        f"{self.profile().sysroot}/usr",
+    ]
```

Comment:
i think this can go at the top level? `self.profile().sysroot` should be available there